introduction/layer-2-payments-on-zksync.md

```diff
@@ -14,7 +14,7 @@ In that previous version - dubbed Brass and later Clay - we implemented batch pa
 
 ## The solution - zkSync
 
-It all changes with the new Golem. That's because the new Golem uses our new, upgraded, ERC-20-compliant token, which, by the way, all current GNT holders already have an ability to migrate to, thanks to the recently-enabled migration process.
+It all changes with the new Golem. That's because the new Golem uses our new, upgraded, ERC-20-compliant token, which, by the way, all current GLM holders already have an ability to migrate to, thanks to the recently-enabled migration process.
```

Comment:
this needs some re-phrasing, the text was clearly made before the GLM token was there.

```suggestion
It all changes with the new Golem. That's because the new Golem uses our new, upgraded, ERC-20-compliant token: GLM. Which, by the way, all current GNT holders already have an ability to migrate to, thanks to the recently-enabled migration process.
```

Reply:
:face-palm: sorry ;) stupid mistake